floris/tools/floris_interface.py

```diff
@@ -208,6 +198,13 @@ def reinitialize(
 
         # Make the given changes
 
+        # First check if wind data is not None,
+        # if not, get wind speeds, wind direction and
+        # turbulence intensity using the unpack_for_reinitialize
+        # method
+        if wind_data is not None:
+            wind_directions, wind_speeds, turbulence_intensity = wind_data.unpack_for_reinitialize()
```

Comment:
What is the expected behavior if `wind_data` and `wind_direction` are both not None?

Reply:
Maybe it's best we throw an error in this case, as it is wind_data is used,

Reply:
how is this @rafmudaf?

Reply:
This is addressed in the next pull request, #782